dark/views/tournament/tround/add.py

```diff
@@ -18,3 +19,12 @@ def get_success_url(self):
             'tournament': self.object.tournament.id,
             'tround': self.object.id
         })
+
+    def post(self, request, *args, **kwargs):
+        if "cancel" in request.POST:
+            tournament_id = kwargs.get('tournament')
+            return redirect(reverse('tournament:info', kwargs={
+                'tournament': tournament_id
+            }))
+        else:
+            super(self)
```

Comment:
Nie jestem pewny, ale to może trzeba przekazać więcej parametrów do super(self)

Reply:
Aye, jutro naprawię

Reply:
Naprawione